pkg/apply/prune/inventory.go

```diff
@@ -93,7 +95,7 @@ func (is *Inventory) Subtract(other *Inventory) (*Inventory, error) {
 	return result, nil
 }
 
-// Equals returns true if the "other" inventory set is the same
+// EqualsWithNormalize returns true if the "other" inventory set is the same
```

Comment:
This comment is different than the name of the function.

Reply:
This is the goland refactoring command that has been too eager. I have updated it.